modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm

```diff
@@ -0,0 +1,557 @@
+
+    full_repbase_logic_name  => "repeatmask_repbase_".$self->o('repbase_logic_name'),
+    repeatmodeler_logic_name => 'repeatmask_repeatmodeler',
+
+    ensembl_analysis_script => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    sequence_dump_script    => catfile($self->o('ensembl_analysis_script'), 'sequence_dump.pl'),
+
+
+########################
+# Extra db settings
+########################
+    num_tokens => 10,
+
+
+########################
+# Executable paths
+########################
+    samtools_path => catfile($self->o('binary_base'), 'samtools'), #You may need to specify the full path to the samtools binary
+
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+########################################################
+# URLs for retrieving the INSDC contigs and RefSeq files
+########################################################
+    ncbi_base_ftp           => 'ftp://ftp.ncbi.nlm.nih.gov/genomes/all',
+    insdc_base_ftp          => $self->o('ncbi_base_ftp').'/#expr(substr(#assembly_accession#, 0, 3))expr#/#expr(substr(#assembly_accession#, 4, 3))expr#/#expr(substr(#assembly_accession#, 7, 3))expr#/#expr(substr(#assembly_accession#, 10, 3))expr#/#assembly_accession#_#assembly_name#',
+    assembly_ftp_path       => $self->o('insdc_base_ftp'),
+    refseq_base_ftp         => $self->o('ncbi_base_ftp').'/#expr(substr(#assembly_refseq_accession#, 0, 3))expr#/#expr(substr(#assembly_refseq_accession#, 4, 3))expr#/#expr(substr(#assembly_refseq_accession#, 7, 3))expr#/#expr(substr(#assembly_refseq_accession#, 10, 3))expr#/#assembly_refseq_accession#_#assembly_name#',
+    refseq_import_ftp_path  => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_genomic.gff.gz',
+    refseq_mrna_ftp_path    => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_rna.fna.gz',
+    refseq_report_ftp_path => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_assembly_report.txt',
+
+
+########################
+# db info
+########################
+    reference_db => {
+      -dbname => $self->o('reference_db_name'),
+      -host   => $self->o('reference_db_server'),
+      -port   => $self->o('reference_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    production_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ensembl_production',
+      -driver => $self->o('hive_driver'),
+    },
+
+    taxonomy_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ncbi_taxonomy',
```

Comment:
```suggestion
      -dbname => 'ncbi_taxonomy',
```

Should there be a variable for the "dbname" in the same way as for the other db details?